modules/creating-infra-machines.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="creating-infra-machines_{context}"]
+= Creating infra machines
+
+if you need infra machines to have dedicated configurations, then you must create an infra pool.
```

Comment:
if -> If

Reply:
Also, the procedure is titled as creating infra machines, but then this intro sentence says that you have to create an infra pool. Not sure if that's all correct, or if there needs to be any adjustment in the title/intro sentence to match.